### ISSUE TITLE
"Request to Restore the Previous Version of LuaDec"

### ISSUE THREAD

**Issue by riyanurpradana** [metaworm/luac-parser-rs#62]
"I kindly request to bring back the previous version of LuaDec, before the recent issues occurred on the site. The current version feels less stable and frequently generates errors, which significantly disrupts my workflow. The earlier version was much better, more stable, and greatly simplified the parsing process. I sincerely hope the previous version can be restored, or at least the issues in the current version are resolved soon. Thank you for your attention, and I hope this can be addressed promptly."

**Comment by metaworm** [comment]
Yes, this update accidentally used an unstable version I wrote before, and I have now switched to the previous version.
